app/assess/models/flag_v2.py

```diff
@@ -36,6 +35,13 @@ def __post_init__(self):
             self.updates[0]["date_created"] if self.updates else ""
         )
 
+    def get_enum_status(self, status):
+        return (
+            FlagTypeV2(status)
+            if isinstance(status, int)
+            else (FlagTypeV2[status] if isinstance(status, str) else status)
+        )
+
```

Comment:
I think this would look better as different lines rather than a single long ternary esque if block

```suggestion
    def get_enum_status(self, status):
        if isinstance(status, int):
            return FlagTypeV2(status)
        elif isinstance(status, str):
            return FlagTypeV2[status]
        return status
```